games/minimal/mods/initial_message/init.lua

```diff
@@ -0,0 +1,6 @@
+minetest.register_on_joinplayer(function(player)
+	local cb = function(player)
+		minetest.chat_send_player(player:get_player_name(), "This is the \"Minimal development test\" [minimal], meant only for testing and development. Use Minetest Game for the real thing.")
```

Comment:
This will crash if the player leaves within two seconds. Although, minimal does tend to only be used in singleplayer :shrug: